src/features/wallet/root/BalanceCard/CurrencyBadge.tsx

```diff
@@ -11,29 +11,27 @@ type Props = {
   amount?: number
 }
 
-const CurrencyBadge = ({ variant, amount }: Props) => (
-  <Box
-    flexDirection="row"
-    backgroundColor="purple300"
-    borderRadius="round"
-    alignItems="center"
-    justifyContent="center"
-    paddingHorizontal="s"
-    height={30}
-    marginRight="m"
-  >
-    {variant === 'dc' && <DC height={16} />}
-    {variant === 'hst' && <HST width={16} />}
-    {variant === 'stake' && <STAKE width={16} color="#A667F6" />}
-    <Text
-      color="white"
-      marginLeft="xs"
-      fontSize={14}
-      visible={amount !== undefined}
+const CurrencyBadge = ({ variant, amount }: Props) => {
+  if (amount === undefined) return null
+  return (
+    <Box
+      flexDirection="row"
+      backgroundColor="purple300"
+      borderRadius="round"
+      alignItems="center"
+      justifyContent="center"
+      paddingHorizontal="s"
+      height={30}
+      marginRight="m"
     >
-      {amount?.toLocaleString(locale)}
-    </Text>
-  </Box>
-)
+      {variant === 'dc' && <DC height={16} />}
+      {variant === 'hst' && <HST width={16} />}
+      {variant === 'stake' && <STAKE width={16} color="#A667F6" />}
```

Comment:
Is this a color we can pull from the theme file?

Reply:
Probably, I have another branch with the mobile token changes so I can clean it up in that one.